clientconn.go

```diff
@@ -117,6 +117,15 @@ type dialOptions struct {
 	disableServiceConfig bool
 }
 
+func defaultDialOptions() dialOptions {
+	return dialOptions{
+		copts: transport.ConnectOptions{
+			WriteBufferSize: -1,
```

Comment:
Is it possible to use `defaultWriteBufSize` instead (and move the const into the grpc package instead of transport)?